assets/css/_icons.scss

```diff
@@ -0,0 +1,13 @@
+.icon {
+  background-repeat: no-repeat;
```

Comment:
Properties should be ordered display, width, height, vertical-align, background-repeat

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)